site/index.html

```diff
@@ -99,6 +100,21 @@
               next(html);
             });
 
+            // Adds the external link target (_blank) to all external links
+            function defaultLinkTargets() {

```

Comment:
The function is directly borrowed from kpt, https://github.com/GoogleContainerTools/kpt/blob/v1.0.0-beta.23/site/static/js/plugins/plugins.js#L89-L101.